### ISSUE TITLE
Possible to add kernel vulnerability tutorial?

### ISSUE THREAD

**Issue by XYlearn** [ctf-wiki/ctf-wiki#254]
I notice that ctf-wiki is short of introduction to kernel vulnerability, but there are many kernel pwn problems in CTF competitions. 

For example
  - **core** and **solid_core** in 2018强网杯
  - **zerofs** in 2018 0ctf

I'm lacking in relative knowledge so can't contribute immediately. Hope someone start the work.

**Comment by iromise** [comment]
Yes, it is on the plan. But it may takes time as most of us is busy..  see https://github.com/ctf-wiki/ctf-wiki/projects/3
